smaps/src/main/java/com/google/smaps/FileParser.java

```diff
@@ -25,6 +25,8 @@
 
 /**
  * Takes in an smap dump text file and produces a list of the regions of that file. {@link Region}
```

Comment:
This should read:
Takes in an smaps dump text file and produces a list of the {@link Region} objects of that file.